commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -164,6 +165,7 @@ private static class NetworkCliCompletionCandidates extends ArrayList<String> {
   @Mixin private PicoKeystoresParameters keystoreParameters;
   @Mixin private PicoCliAwsSecretsManagerParameters awsSecretsManagerParameters;
   @Mixin private PicoCliGcpSecretManagerParameters gcpSecretManagerParameters;
+  @Mixin private PicoCommitBoostApiParameters commitBoostApiParameters;
```

Comment:
It's not quite right to add this into the eth2 API as it has secp signing as well. I think it would be better to add this to the default or base command rather than a subcommand as that's where we want all the other functionality eventually too

Reply:
Because the upcoming GenerateProxyKey requires Spec (to calculate genesis fork version) and gvr to compute the root. The spec and gvr is specific to "eth2" mode, hence the commit boost API is specific to eth2 mode as well and won't work in eth1 mode at all. 

I can still move it to base level, but we will only be using it in eth2 mode.